src/itkOMEZarrNGFFImageIO.cxx

```diff
@@ -341,16 +349,17 @@ OMEZarrNGFFImageIO::ReadImageInformation()
   bool status = jsonRead(std::string(this->GetFileName()) + "/.zgroup", json);
   assert(json.at("zarr_format").get<int>() == 2); // only v2 for now
   status = jsonRead(std::string(this->GetFileName()) + "/.zattrs", json);
-  json = json.at("multiscales")[0];    // multiscales must be present in OME-NGFF
-  assert(json.at("version") == "0.4"); // only this is currently supported
-  this->SetNumberOfDimensions(json.at("axes").size());
+  json = json.at("multiscales")[0]; // multiscales must be present in OME-NGFF
+  auto version = json.at("version").get<std::string>();
+  assert(version == "0.4" || version == "0.3" || version == "0.2" || version == "0.1"); // supported versions
```

Comment:
Is it possible that revision numbers could be included (i.e. "0.4.1', "0.3.4", etc)? If so then that is another case to handle.

Reply:
I will just remove this assert. That way even a future version file could be partly read.